NetExericise/NetExericise/Basic/Exercise09.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetExericise.Basic
+{
+    public class Exercise09
+    {
+        public double CalculateSqrt(double n, double e)
+        {
+            if (n == 1) return 1;
+
+            double left, right, mid;
```

Comment:
`left = 0` trong cả 2 trường hợp, nên để ra ngoài. `right` có thể dùng toán tử `?:` để gán giá trị

Reply:
Done